### DISCUSSION TITLE
projection of sulcus on Freesurfer's pial

### DISCUSSION THREAD

**Discussion by iamjoostjanssen** [brainvisa/brainvisa.github.io#142] [General]
Hi,

I wonder if any of you projected the labelled Morphologist-sulci (default outcome of Morphologist pipeline) on Freesurfer's fsaverage surface? I think what I'm asking if a freesurfer fsaverage 'annot' exists of the Morphologist-produced sulci. I hope I'm making sense, apologies if I do not.

Merci,
-joost

**Comment by denisri** [comment]
Not directly, because 1. Morphologist surfaces are not Freesurfer ones, 2. sulci are not extracted as lines but as 3D objects in voxel-based volumes. The projection of sulci on the cortical surface is not an obvious operation, but there is such kind of projection in the HipHop pipeline in the cortical surface toolbox. To use it for freesurfer surfaces, you can probably import FS surfaces using the brainvisa/freesurfer toolbox (freesurfer outputs to brainvisa conversion, taking care of the mesh resampling you want), then project using the process from HipHop. The output texture can normally be mapped on the fsaverage with the same resampling.
I have not tried this procedure so I can't swear it works, but it's an idea...
Denis

**Comment by gauzias** [comment]
Hi,
You might have a look at our paper introducing HipHop https://ieeexplore.ieee.org/abstract/document/6416068
in which we did project sulci from brainvisa onto individual surfaces from freesurfer and then map these onto fsaverage.
This does not correspond to projecting sulci onto fsaverage directly, and that is my point: when mapping sulci from individuals onto an average brain, you can observe the residual dispersion remaining despite non-rigid registration.
This residual dispersion questions the meaning of tracing sulci on average surface like fsaverage.

**Comment by iamjoostjanssen** [comment]
Hi,

Thank you both for your comments. The reason I asked this question is because I have several digital atlasses (e.g. von Economo) in fsaverage space and I'd like to know for brainvisa labelled sulci which atlas-regions these sulci traverse. Do you guys know if anybody looked at this? 

Merci,
-joost
